src/pages/StatisticsChart/activities/CkbHodlWave.tsx

```diff
@@ -0,0 +1,326 @@
+  }): string => {
+    if (seriesName === t('statistic.latest_day')) {
+      return `<div>${tooltipColor(color)}${widthSpan(t('statistic.latest_day'), currentLanguage)} ${handleAxis(
+        data[1],
+        2,
+      )}% </div>`
+    }
+    if (seriesName === t('statistic.day_to_one_week')) {
+      return `<div>${tooltipColor(color)}${widthSpan(t('statistic.day_to_one_week'), currentLanguage)} ${handleAxis(
+        data[2],
+        2,
+      )}</div>`
+    }
+    if (seriesName === t('statistic.one_week_to_one_month')) {
+      return `<div>${tooltipColor(color)}${widthSpan(
+        t('statistic.one_week_to_one_month'),
+        currentLanguage,
+      )} ${handleAxis(data[3], 2)}%</div>`
+    }
+    if (seriesName === t('statistic.one_month_to_three_months')) {
+      return `<div>${tooltipColor(color)}${widthSpan(
+        t('statistic.one_month_to_three_months'),
+        currentLanguage,
+      )} ${handleAxis(data[4], 2)}%</div>`
+    }
+    if (seriesName === t('statistic.three_months_to_six_months')) {
+      return `<div>${tooltipColor(color)}${widthSpan(
+        t('statistic.three_months_to_six_months'),
+        currentLanguage,
+      )} ${handleAxis(data[5], 2)}%</div>`
+    }
+    if (seriesName === t('statistic.six_months_to_one_year')) {
+      return `<div>${tooltipColor(color)}${widthSpan(
+        t('statistic.six_months_to_one_year'),
+        currentLanguage,
+      )} ${handleAxis(data[6], 2)}%</div>`
+    }
+    if (seriesName === t('statistic.one_year_to_three_years')) {
+      return `<div>${tooltipColor(color)}${widthSpan(
+        t('statistic.one_year_to_three_years'),
+        currentLanguage,
+      )} ${handleAxis(data[7], 2)}%</div>`
+    }
+    if (seriesName === t('statistic.over_three_years')) {
+      return `<div>${tooltipColor(color)}${widthSpan(t('statistic.over_three_years'), currentLanguage)} ${handleAxis(
+        data[8],
+        2,
+      )}%</div>`
+    }
+
+    return ''
+  }
+}
+
+const useOption = (
+  statisticCkbHodlWaves: ChartItem.CkbHodlWave[],
+  chartColor: ChartColorConfig,
+  isMobile: boolean,
+  isThumbnail = false,
+): echarts.EChartOption => {
+  const { t } = useTranslation()
+  const currentLanguage = useCurrentLanguage()
+  const gridThumbnail = {
+    left: '4%',
+    right: '10%',
+    top: '8%',
+    bottom: '6%',
+    containLabel: true,
+  }
+  const grid = {
+    left: '3%',
+    right: '3%',
+    top: '8%',
+    bottom: '5%',
+    containLabel: true,
+  }
+  const parseTooltip = useTooltip()
+  return {
+    color: chartColor.colors,
+    tooltip: !isThumbnail
+      ? {
+          trigger: 'axis',
+          formatter: dataList => {
+            assertIsArray(dataList)
+            let result = `<div>${tooltipColor('#333333')}${widthSpan(t('statistic.date'), currentLanguage)}
+          ${dataList[0].data[0]}</div>`
+            dataList.forEach(data => {
+              assertSerialsItem(data)
+              assertSerialsDataIsStringArrayOf9(data)
```

Comment:
It's better to wrap them in a `try catch` block to prevent from page crash when the data is invalid.